### ISSUE TITLE
"TypeError" exception caused by "Numpy" 1.18 in "colour.SpectralShape.range" method.

### ISSUE THREAD

**Issue by bersbersbers** [colour-science/colour#540]
```
$ pip install "numpy<1.18.0"
...
$ python -c "from colour import XYZ_to_sRGB"
(works fine)

$ pip install numpy==1.18.0
...
$ python -c "from colour import XYZ_to_sRGB"
TypeError: object of type <class 'numpy.float64'> cannot be safely interpreted as an integer.
```

**Comment by KelSolaar** [comment]
Hi @bersbersbers,

This should be fixed in [develop](https://github.com/colour-science/colour/tree/develop) as per #536.

Cheers,

Thomas

**Comment by bersbersbers** [comment]
And I believe this has not shipped in `0.3.15`, thanks!

**Comment by KelSolaar** [comment]
Yes it should be! :)
